~/.polykey/.polykey/metadata

```diff
@@ -0,0 +1 @@
+{"vaults":{},"publicKeyPath":null,"privateKeyPath":null,"passphrase":null}
```

Comment:
What's this file?

Reply:
this was a file left in by mistake, it h as been removed